CHANGELOG.md

```diff
@@ -26,14 +30,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - The copyright for all sources is now attributed to Open Enclave SDK contributors.
 - Update Intel DCAP library dependencies to 1.4.1.
 - Update Intel PSW dependencies to 2.6.100.2 on Windows.
-- The stack guard for OE SDK on SGX is turned on. It is also turned on for enclaves.
+- Enable /W2 to treat W2 warnings as errors for Windows builds. Additionally, fix errors which arise from enabling this flag.
+- Fix oe_random for size > 1024.
```

Comment:
Keepachangelog.com guidance is to have a separate __Fixed__ section for bug fixes of note (see previous version notes in this file)

```
### Fixed
- `oe_random()` now correctly returns a fully filled byte buffer for requests of > 1024 bytes.
- Add `openenclave` namespace to dl and crypto libraries to prevent symbol collisions. Fixes #2082. 
```